src/ios/CDVWKInAppBrowser.m

```diff
@@ -1289,6 +1291,18 @@ - (UIColor *)colorFromHexString:(NSString *)hexString {
     return [UIColor colorWithRed:((rgbValue & 0xFF0000) >> 16)/255.0 green:((rgbValue & 0xFF00) >> 8)/255.0 blue:(rgbValue & 0xFF)/255.0 alpha:1.0];
 }
 
+// Helper function to return a slightly darker version of UIColor
+- (UIColor *)darkerColorForColor:(UIColor *)c
+{
+    CGFloat r, g, b, a;
+    if ([c getRed:&r green:&g blue:&b alpha:&a])
+        return [UIColor colorWithRed:MAX(r - 0.2, 0.0)
+                               green:MAX(g - 0.2, 0.0)
+                                blue:MAX(b - 0.2, 0.0)
+                               alpha:a];
+    return nil;
+}
```

Comment:
Similar. This one is returning a darker shade and `manipulate` returns a darker or lighter shade depending on `factor`. Maybe I should make them do the same thing so it's more consistent 🤔

Reply:
I think it would be useful to be as consistent as possible between the two. 

I would advocate for removing manipulate if we can get away with just adding the darker shade. If we cannot just darken, `manipulate` is probably fine

Reply:
I changed `manipulateColor` to `darkerColorForColor` for Android.